plugins/scaffolder/src/components/TemplateEditorPage/DryRunContext.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('base64EncodeContent', () => {
   it('encodes text files', () => {
     expect(base64EncodeContent('abc')).toBe('YWJj');
     expect(base64EncodeContent('abc'.repeat(1000000))).toBe(
-      btoa('<file too large>'),
+      Buffer.from('<file too large>').toString('base64'),
```

Comment:
This one is a bit tricky tbh, because we've got type from multiple TypeScript libraries competing here. The deprecation is coming from the Node.js types, but this is web code, so doesn't really apply here. The workaround that you can use for all of the usages of `btoa(...)` here is to use `window.btoa(...)` instead.